pkg/operator/ceph/object/bucket/s3-handlers.go

```diff
@@ -176,15 +176,14 @@ const (
 )
 
 // AllowedActions is a lenient default list of actions
-var AllowedActions = []action{
+var OBCAllowedActions = []action{
 	DeleteObject,
 	DeleteObjectVersion,
 	GetBucketAcl,
 	GetBucketCORS,
 	GetBucketLocation,
 	GetBucketLogging,
 	GetBucketNotification,
-	GetBucketPolicy,
```

Comment:
How is this related to the main fix? It feels like it should be its own commit unless I'm missing something.

Reply:
> How is this related to the main fix? It feels like it should be its own commit unless I'm missing something.

Yeah it is not needed, will change in next update

Reply:
> > How is this related to the main fix? It feels like it should be its own commit unless I'm missing something.
> 
> Yeah it is not needed, will change in next update

@leseb sorry this change is needed. Otherwise I may need to rewrite actions for the policy in this pr

Reply:
It's fine, just explain why it's needed then :)